test/e2e/shared/cluster.go

```diff
@@ -53,7 +53,7 @@ func createClusterctlLocalRepository(config *clusterctl.E2EConfig, repositoryFol
 	createRepositoryInput.RegisterClusterResourceSetConfigMapTransformation(ccmPath, CCMResources)
 
 	clusterctlConfig := clusterctl.CreateRepository(context.TODO(), createRepositoryInput)
-	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl config file does not exists in the local repository %s", repositoryFolder)
+	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctlConfig file does not exists in the local repository %s", repositoryFolder)
```

Comment:
WDYT about

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file %q does not exists in the local repository %s", clusterctlConfig, repositoryFolder)
```

or

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file does not exists in the local repository %s", repositoryFolder)
```

Tbh, I don't like variable names in log output :)

Reply:
Another solution could be

```suggestion
	Expect(clusterctlConfig).To(BeAnExistingFile(), "The clusterctl generate file %q does not exists", clusterctlConfig)
```

Reply:
@obiasgiese
The `The clusterctl config file does not ....` is a typo. It was referring to the file mentioned in the `Expect(....)`. 
I was just fixing that. Not related to the actual `clusterctl config `command.

Reply:
Ah, sure. Didn't pay attention to the context. Thanks for clarification ;) 👍🏻